bitnami/kafka/values.yaml

```
@@ -348,6 +348,11 @@ interBrokerListenerName: INTERNAL
 ##
 replicaCount: 1
 
+## Minimal broker.id value
+## Brokers increment their ID starting at this minimal value.
+## E.g., with `minBrokerId=100` and 3 nodes, IDs will be 100, 101, 102 for brokers 0, 1, and 2, respectively.
+##
+
```

`minBrokerId` should be set to `0` by default:
```suggestion
minBrokerId: 0
```